packages/trip-form/src/MetroModeSelector/AdvancedModeSettingsButton/index.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import AnimateHeight from "react-animate-height";
+import styled from "styled-components";
+import colors from "@opentripplanner/building-blocks";
+import { Check2 } from "@styled-icons/bootstrap";
+import { ModeButtonDefinition } from "@opentripplanner/types";
+import { useIntl } from "react-intl";
+import SubSettingsPane from "../SubSettingsPane";
+import generateModeButtonLabel from "../i18n";
+import { invisibleCss } from "..";
```

Comment:
I think there should be a an empty line between the import groups here?

Reply:
I'm not getting any errors or warnings here, but it looks like there should be right?